readthedocs/templates/builds/build_detail.html

```diff
@@ -53,6 +53,16 @@
           {{ build.builder }}
         </li>
       {% endif %}
+
+      {% if request.user.is_staff %}
```

Comment:
Is there a reason we're only showing this to admin users? Seems like it's useful enough that it should be exposed to at least the project owners, if not anyone.